### PR TITLE
feat: add extra container for sidecar uploader

### LINKED ISSUE #919: Record stream files under sidecar directory is not uploaded
### Bill of Materials or CLI Version

_No response_

### Execution Environment

JUnit Extensions

### Slf4j Logging Implementation

None

### To Reproduce

```markdown
Run current helm chart
```

### Describe the bug

Files under /opt/hgcapp/recordStreams/record0.0.x/sidebar directory were not uploaded to minio storage

### Describe the expected behavior

Should upload sidecar file too

### Whole JUnit/CLI Logs

```shell
N/A
```

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

- Add another pod for hosting sidecar uploader

### Related Issues

- Closes #919

### CODE REVIEW

File: charts/fullstack-deployment/values.yaml [hunk 1]
```diff
@@ -115,6 +116,27 @@ defaults:
...
+        compression: true
+        sidecar: true
+        reaper:
+          enabled: true
+          minKeep: 1
+          interval: 1
+          defaultBackoff: 1
+        signature:
+          require: true
+          prioritize: true
```
Comment (leninmehedy) [inline charts/fullstack-deployment/values.yaml:138]:
```suggestion
        signature:
          require: false
          prioritize: false
```
Based on Ivo's comment: https://swirldslabs.slack.com/archives/C06FWU24B7Y/p1720445868302879

File: charts/fullstack-deployment/templates/network-node-statefulset.yaml [hunk 2]
```diff
@@ -341,6 +346,68 @@ spec:
...
+          - name: RECORD_STREAM_SIDECAR
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.sidecar ($recordStreamSidecar.config).sidecar | quote }}
+          - name: SIG_REQUIRE
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.signature.require (($recordStreamSidecar.config).signature).require | quote }}
+          - name: SIG_PRIORITIZE
+            value: {{ default $defaults.sidecars.recordStreamSidecarUploader.config.signature.prioritize (($recordStreamSidecar.config).signature).prioritize | quote }}
+          - name: BUCKET_PATH
+            value: "recordstreams/record{{ $node.accountId }}"
+          - name: BUCKET_NAME
+            value: {{ $cloud.buckets.streamBucket | quote }}
```
Comment (leninmehedy) [inline charts/fullstack-deployment/templates/network-node-statefulset.yaml:398]:
```suggestion
          - name: BUCKET_PATH
            value: "recordstreams/record{{ $node.accountId }}/sidecar"
          - name: BUCKET_NAME
            value: {{ $cloud.buckets.streamBucket | quote }}
```
Based on Ivo's comment: https://swirldslabs.slack.com/archives/C06FWU24B7Y/p1720445868302879

File: charts/fullstack-deployment/values.yaml (hunk 1 above)
Comment (leninmehedy) [reply charts/fullstack-deployment/values.yaml:138]: I've checked the bucket and it does have a sidecar directory like `.../recordstreams/record0.0.10/sidecar` and it does not contain the signature file.

**Review by Ivo-Yankov — CHANGES_REQUESTED** [review]
Looks good. Can you also add one more change to `fullstack-deployment/values.yaml`:  Set this value `hedera.mirror.importer.parser.record.sidecar.enabled=true`. This will enable the importing of sidecar records to the mirror node.

File: charts/fullstack-deployment/templates/network-node-statefulset.yaml (hunk 2 above)
Comment (JeffreyDallas) [reply charts/fullstack-deployment/templates/network-node-statefulset.yaml]: Fixed

File: charts/fullstack-deployment/values.yaml (hunk 1 above)
Comment (JeffreyDallas) [reply charts/fullstack-deployment/values.yaml]: Fixed

**Review by Ivo-Yankov — APPROVED** [review]
LGTM

**Pull request merged**